common/test/acceptance/fixtures/__init__.py

```diff
@@ -1,11 +1,15 @@
 import os
 
 
+HOSTNAME = os.environ.get('BOK_CHOY_HOSTNAME', 'localhost')
+CMS_PORT = os.environ.get('BOK_CHOY_CMS_PORT', '8031')
```

Comment:
And to be clear, I do not think we should create another ticket or worry about the fact that 8031 is what we are using.  I was simply saying we might want to add a comment here that the port must be 8031 because it is hardcoded elsewhere.

In any case, I'd leave this as-is and move on.  Thanks.  Sorry about the confusion.